src/codegen/sys/lib_.rs

```diff
@@ -444,7 +417,7 @@ fn generate_records(w: &mut Write, env: &Env, records: &[&Record]) -> Result<()>
         if lines.is_empty() {
             try!(writeln!(
                 w,
-                "{}#[repr(C)]\n{0}pub struct {}(c_void);\n",
+                "{0}#[repr(C)]\n{0}#[derive(Copy,Clone)]\n{0}pub struct {1}(u8);\n",
```

Comment:
`extern type` is that we need for this case but it not realized and may cause build error when inserted in other struct directly (as with `GIConv`)
`#[repr(transparent)]` can be used for minimize alignment errors for known fields, so also not for this case, so last `repr(C)`.
I prefer `c_void` over `u8` as it don't implement `Copy` while has same size and alignment.